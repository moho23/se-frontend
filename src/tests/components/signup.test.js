import React from "react";
import Signup from "../../MainProject/register/signup/signup.index"
import {mount, shallow} from "enzyme";

test('should render Signup correctly', () => {
    const wrapper = shallow(<Signup />);
    expect(wrapper).toMatchSnapshot();
  });


test("rendering Signup component <P1> whithout crashing",() =>{
    const wrapper=shallow(<Signup/>);
    const p1 =(<p>خوش اومدی :)</p>);
    expect(wrapper.contains(p1)).toEqual(true);
})