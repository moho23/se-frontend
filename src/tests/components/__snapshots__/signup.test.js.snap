// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Signup correctly 1`] = `ShallowWrapper {}`;
